File: fix_line.js
const fs = require('fs');
let lines = fs.readFileSync('src/components/RfqActions.tsx', 'utf8').split('\n');
lines[78] = '        subject: \Purchase Order - \ from \\,';
fs.writeFileSync('src/components/RfqActions.tsx', lines.join('\n'));

File: fix_line.py

with open('src/components/RfqActions.tsx', 'r', encoding='utf-8') as f:
    lines = f.read().split('\n')

lines[78] = '        subject: \Purchase Order - \ from \\,'

with open('src/components/RfqActions.tsx', 'w', encoding='utf-8') as f:
    f.write('\n'.join(lines))

